README.md

```diff
@@ -74,31 +83,8 @@ Contributing
  7. Create a new Pull Request
 
 
-Boring legal stuff
+Links
 ------------------
 
-> Copyright © 2015 Andrey Novikov
->
-> MIT License
->
-> Permission is hereby granted, free of charge, to any person obtaining
-> a copy of this software and associated documentation files (the
-> "Software"), to deal in the Software without restriction, including
-> without limitation the rights to use, copy, modify, merge, publish,
-> distribute, sublicense, and/or sell copies of the Software, and to
-> permit persons to whom the Software is furnished to do so, subject to
-> the following conditions:
->
-> The above copyright notice and this permission notice shall be
-> included in all copies or substantial portions of the Software.
->
-> THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-> EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-> MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-> NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
-> LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
-> OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
-> WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
-[PgTune]: http://pgtune.leopard.in.ua/
-[librarian-puppet]: http://librarian-puppet.com/
+ * [PgTune]: http://pgtune.leopard.in.ua/
+ * [librarian-puppet]: http://librarian-puppet.com/
```

Comment:
GitHub's Markdown doesn't understand [link reference definitions](http://spec.commonmark.org/0.26/#link-reference-definitions) in a lists, so first paragraphs are broken.

Please just remove the **Links** heading and return definitions to previous format. They are not needed to be displayed on page:

```markdown
[PgTune]: http://pgtune.leopard.in.ua/
[librarian-puppet]: http://librarian-puppet.com/
```